src/Microsoft.AspNet.Mvc.Core/Resources.resx

```diff
@@ -412,4 +412,7 @@
   <data name="SerializableError_DefaultError" xml:space="preserve">
     <value>The input was not valid.</value>
   </data>
+  <data name="ResponseCache_SpecifyDuration" xml:space="preserve">
+    <value>Duration parameter must be specified.</value>
```

Comment:
This needs to be improved. For one `Duration` is a property. Also, we don't really need to set it when `NoStore = true`